src/models/event.js

```diff
@@ -747,7 +741,9 @@ utils.extend(module.exports.MatrixEvent.prototype, {
      * @return {boolean}
      */
     isRelation(relType = undefined) {
-        const content = this.getContent();
+        // must use event.content as m.relates_to is not encrypted
+        // and _clearEvent doesn't have it.
+        const content = this.event.content;
```

Comment:
Locally I have added a `getRelation` helper and replaced all accesses of `m.relates_to` to use that, so I'll apply the same cleanup to these bits once they land.